CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [UNRELEASED] - 2018-12-19
+### Fixed
+[#110](https://github.com/plotly/dash-renderer/pull/110)
+
+- Keep the config store state on soft reload.
+- AppProvider returns `Loading...` if no configs as before #108
+
```

Comment:
Merge all unreleased changes into the same block, right? Since once we *do* make a new release, they will all be part of it.

Reply:
I wanted to have the timestamp with the changes. It does look weird, will change.